cordova-plugin-outline/android/java/org/outline/shadowsocks/Shadowsocks.java

```diff
@@ -51,6 +52,7 @@ public synchronized boolean start(JSONObject serverConfig) throws JSONException
         "-b", LOCAL_SERVER_ADDRESS,
         "-l", LOCAL_SERVER_PORT,
         "-m", serverConfig.getString("method"),
+        "-t", String.format(Locale.ROOT, "%d", SS_LOCAL_TIMEOUT_SECS),
```

Comment:
Why do we need String.format? We just need to convert to the number in ascii. Won't we get a localized String in some languages (arabic, farsi, ...)